torch/distributed/internal_rpc_utils.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pickle
+import copyreg
+import io
+import collections
+
+def serialize(obj):
+    f = io.BytesIO()
+    p = pickle.Pickler(f)
+    p.dispatch_table = copyreg.dispatch_table.copy()
+    p.dump(obj)
+    return f.getvalue()
+
+def run_python_udf_internal(pickled_python_udf):
+    try:
+        python_udf = pickle.loads(pickled_python_udf)
+        result = python_udf.func(*python_udf.args, **python_udf.kwargs)
+    except Exception as e:
+        result = "run_python_udf_internal caught exception: " + str(e)
```

Comment:
This is the line that silently swallows Exceptions

Reply:
@xush6528 I add this exception in python land and pass it back to client in this way, because we do not have general remote exception handling yet. if we do not pass it in this way and there is function call exception here, server will crash. once we have general remote exception, we can have a new type of message and make it more clear to users. cc @mrshenli  does it make sense for error handling?

Reply:
so right now users need to know how to validate result is expected or exception, before we add remote exception handling

Reply:
Yes, that makes sense to me. And I think it can come in a followup PR, right?

Reply:
@zhaojuanmao 
Thanks for clarifying the plan.

So, in recap, the plans is that,
we will have a unified error handling in C++ world that can handle
- built-in op error.
- python function error.

So leave this workaround here for now, and improve it later.